eth_trie/db.nim

```diff
@@ -0,0 +1,3 @@
+import types
+export types
+
```

Comment:
The contents of the types file will be moved here eventually (after getting rid of `eth_trie/types` imports in other projects).